build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Not sure we need this

Reply:
I've been running this to do api development locally